docs/user/FAQS.md

```diff
@@ -27,19 +27,31 @@ git pull
 Now your local copy of ORFS should be up-to-date. The next step is to build it again with the following command: 
 
 For WSL/docker based installation, run:
-```
+```shell
 ./build_openroad.sh --clean
 ```
 
 For local installation, run:
-```
+```shell
 ./build_openroad.sh --local --clean
 ```
 
 For pre-built binaries, you can run the flow after `git pull` is done.
 
-> **TIP**: For development purposes, it is a good practice to work on branches and leave master untouched. 
-> To update your branch, simply run `git checkout <BRANCH>; git merge master`
+:::{tip} 
+For development purposes, it is a good practice to work on branches and leave master untouched. 
+- Branch already exists: `git checkout <BRANCH> && git merge master`
+- Branch does not exist: `git checkout -b <BRANCH> && git merge master` 
+:::
+
 ## How do I update the design reference files?
 
-See how to update the Metrics [here](../contrib/Metrics.md).
+See how to update using the Metrics [guide](../contrib/Metrics.md).
+
+## How do I get better search results?
```

Comment:
Let's add a `tip` code block pointing users to this section. Before the "Getting started" section.
![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/9001905/6bbe363f-397c-4a5d-a561-0234f77151ee)
```
TIP: see these tips to help improve your search results. <LINK>
```